### PR TITLE
Add Wombat Exchange (Base) - Ready for review

### PR DESCRIPTION
**Pull request by henrystats**
# Thank you for contributing to Spellbook!

Thank you for taking the time to submit code in Spellbook. A few things to consider:

- If you are a first-time contributor, please sign the CLA by copy & pasting **_exactly_** what the bot mentions in PR comment
- Refer to [docs](#spellbook-contribution-docs) section below to answer questions
- Dune team will review submitted PRs as soon as possible

## Spellbook sub-projects

To prepare for Spellbook to scale in size, Dune team has started to pilot sub-projects:

- For a quick overview, see [the GH discussion on the topic](https://github.com/duneanalytics/spellbook/discussions/5238)
  - Please ask questions and continue the conversation there
- The first spell lineage to move: `tokens.erc20` and all upstream spells which help build it
  - **_Note_**: Please navigate to [tokens](/tokens) root directory to find tokens objects

## Spellbook contribution docs

The [docs](docs) directory has been implemented to answer as many questions as possible. Please take the time to reference each `.md` file within this directory to understand how to efficiently contribute & why the repo is designed as it is 🪄

Example questions to be answered:

- What does each property in the [model config block](/docs/models/model_config_block.md) mean?
- What is the [CI test](/docs/ci_test/ci_test_overview.md) attached to PRs and how can I best utilize it?
- What are the Spellbook [best practices](/docs/general/best_practices.md)?

Please navigate through the [docs](/docs) directory to find as much info as you can.

**Note**: happy to take PRs to improve the docs, let us know 🤝

### CODE REVIEW

**Comment by henrystats** [comment]
@jeff-dude this is now ready for review!

File: models/_sector/dex/trades/base/_schema.yml [hunk 1]
```diff
@@ -626,11 +626,30 @@ models:
       contributors: Henrystats, tomfutago
     config:
       tags: [ 'base', 'dex', 'trades', 'openocean' ]
-    description: "openocean base base trades"
+    description: "leetswap_v2 base base trades"
```
Comment (Hosuke) [inline models/_sector/dex/trades/base/_schema.yml:629]: By the way, what's the relationship between `leetswap` and `openocean`?
Leetswap's schema has already been defined in [Line 602](https://github.com/duneanalytics/spellbook/pull/5429/files#diff-60520bb14954436eba8e725c491507a5165a3f6b8a2a0fcff85af99cf6f7cb1bR602).

I think we can recover this line.

**Review by Hosuke — APPROVED** [review]
LGTM.✅ for wombat_exchange_base_base_trades
Thank you @henrystats 

[Sample wombat_exchange_base_base_trades](https://dune.com/queries/3471643)

File: models/_sector/dex/trades/base/_schema.yml (hunk 1 above)
Comment (henrystats) [reply models/_sector/dex/trades/base/_schema.yml]: @Hosuke it was while I was trying to resolve conflicts the lines got mixed up. Fixed the description

Comment (henrystats) [reply models/_sector/dex/trades/base/_schema.yml]: resolved the conflicts as well @Hosuke

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
we can ignore that error (prod hasn't run yet, so CI can't do lookup)

**Pull request merged**